CommonTools/PileupAlgos/python/PhotonPuppi_cff.py

```diff
@@ -5,7 +5,7 @@
                              candName       = cms.InputTag('packedPFCandidates'),
                              puppiCandName  = cms.InputTag('puppi'),
                              photonName     = cms.InputTag('slimmedPhotons'),
-                             photonId       = cms.InputTag("egmPhotonIDs:cutBasedPhotonID-Spring15-25ns-V1-standalone-loose"), 
+                             photonId       = cms.InputTag("egmPhotonIDs:cutBasedPhotonID-Spring16-V2p2-loose"), 
```

Comment:
puppi is a part of miniAOD.
This PR is supposed to leave the miniAOD unchanged.
Please remove edits in this file.